py/picca/delta_extraction/corrections/ivar_correction.py

```diff
@@ -43,10 +43,8 @@ def __init__(self, config):
             hdu = fitsio.read(filename, ext="VAR_FUNC")
             if "loglam" in hdu.dtype.names:
```

Comment:
Related to the comment above, I'm not sure we need to allow for these different HDU names (unless we need it for compatibility with older delta files)

Reply:
I left this for compatibility with older files. Until this is merged, the picca_delta_extraction saves the attributes files with `lambda` on it. We can add a `DeprecationWarning` and remove after a while

Reply:
Let's leave both options for compatibility.